### PR TITLE
add 4 extra featured projects to explore page (closes #971)

### LINKED ISSUE #971: Add 4 extra featured projects to explore page
So just a second row of four here
image

My suggestions:
- https://www.drips.network/app/projects/github/NiceNode/nice-node
- https://www.drips.network/app/projects/github/starknet-io/starknet.js
- https://www.drips.network/app/projects/github/snapshot-labs/snapshot
- https://www.drips.network/app/projects/github/efstajas/svelte-stepper
- https://www.drips.network/app/projects/github/heyxyz/hey
- https://www.drips.network/app/projects/github/openlittermap/openlittermap-web

### PR DESCRIPTION
**Pull request by SouravInsights**
## Description:
Added `accountIds` of 4 extra projects ([nice-node](https://www.drips.network/app/projects/github/NiceNode/nice-node), [startkenet.js](https://www.drips.network/app/projects/github/starknet-io/starknet.js), [snapshot](https://www.drips.network/app/projects/github/snapshot-labs/snapshot) & [svelte-stepper](https://www.drips.network/app/projects/github/efstajas/svelte-stepper)) to the mainnet array of `FEATURED_PROJECT_ACCOUNT_IDS`.

## Screenshots:
CleanShot 2024-02-22 at 04 48 32@2x

## Note:
I'm not sure about the criteria to select featured projects, but as far as I understand, the consensus is to strike a balance between showcasing well-known projects and giving visibility to lesser-known projects that might deserve attention. Choosing popular projects potentially increase engagement within the platform while promoting newer or less recognized projects, foster diversity & encourage community growth. 

But currently, I have added **Nice Node**, **Snapshot** & **Starknet.js** and all 3 projects are well known so let me know if I need to include any other projects to the list. From a quick glance, I think [ens-normalize.js](https://github.com/adraffy/ens-normalize.js) is a cool one! Haven't explored all other projects as of now... Let me know your thoughts! @brandonhaslegs @efstajas 

closes #971

### CODE REVIEW

**Comment by brandonhaslegs** [comment]
Hey thanks! Someone will take a look today. Can you mention which issue this closes so that it'll close the issue when the PR is merged?

**Comment by SouravInsights** [comment]
> Hey thanks! Someone will take a look today. Can you mention which issue this closes so that it'll close the issue when the PR is merged?

Sure! This is the issue: https://github.com/drips-network/app/issues/971

**Comment by brandonhaslegs** [comment]
Sorry btw I meant something like this:

closes #971

**Comment by SouravInsights** [comment]
> Sorry btw I meant something like this:
> 
> closes #971

oh right right.. sorry!!

File: src/routes/app/(app)/+page.svelte [hunk 1]
```diff
@@ -33,6 +33,10 @@
         '80928956806149918791864723629668437820661066502202314166815319654400',
         '80989205010981758696261160004449877944077887004065826078532843448906',
         '80921140646830818724035150101819719966329403614944137690624336855040',
+        '80912096692731427285642748238291568975884076216206760523486136893440', // nice-node
+        '80927335273972468167722947750338907267861671542981060844246982983680', // startkenet.js
```
Comment (efstajas) [inline src/routes/app/(app)/+page.svelte:37]: comment should be "starknet.js" :)

**Comment by evvvritt** [comment]
thanks! can you please put `closes #971` in the title of this PR so when we squash and merge it will auto-close the issue?

**Comment by brandonhaslegs** [comment]
> thanks! can you please put `closes #971` in the title of this PR so when we squash and merge it will auto-close the issue?

I think it still works as a comment right? We talked about that further up there 👆

File: src/routes/app/(app)/+page.svelte (hunk 1 above)
Comment (jo-elimu) [inline src/routes/app/(app)/+page.svelte:37]:
```suggestion
        '80927335273972468167722947750338907267861671542981060844246982983680', // starknet.js
```

**Comment by evvvritt** [comment]
No I think it needs to be in the title to auto-close (since title is used as the commit message which triggers the auto-close) @brandonhaslegs

**Comment by evvvritt** [comment]
Code correct but is stacking 8 projects on mobile too tall? @brandonhaslegs

**Comment by SouravInsights** [comment]
> Code correct but is stacking 8 projects on mobile too tall? @brandonhaslegs

I can think of 2 ideas..

- **Pagination:** We can split the 8 cards into smaller subsets, likes 4 cards per page and have controls to navigate between pages.

- **Carousel:** We can also have an horizontal carousel that allows users to swipe left or right to view cards. Although it's still a long list to scroll through but I think users are accustomed to swiping left or right to navigate through content on mobile devices, which makes horizontal scrolling feel natural and intuitive.

**Comment by brandonhaslegs** [comment]
Good point. I think a horizontal overflow would be fine in mobile but need to design something. You can also suggest something directly in code if you'd like.

**Comment by efstajas** [comment]
> No I think it needs to be in the title to auto-close (since title is used as the commit message which triggers the auto-close) @brandonhaslegs

it doesn't actually, just needs to be in the initial PR description. just added it there :) 

> No no, that's not what I meant! I mean we should definitely give visibility to lesser-known projects by individuals too. If all the featured projects are well known & popular then it would be not as diverse & maybe a bit misaligned. Bdw, I just checked out svelte-stepper & I think it's a cool project that deserve attention!

I got that, all good! I just think graph node should be featured for sure generally (because it's a big name), and am not super sure that featuring a project by one of the core maintainers of Drips is necessarily the best look, though I really appreciate that you like it 🙌 

> Code correct but is stacking 8 projects on mobile too tall?

... We could also just hide everything after the first 4 projects on mobile with a media query. Maybe a bit strange to have different content depending on screen size but I think in this instance it could be OK.

**Comment by efstajas** [comment]
> No I think it needs to be in the title to auto-close (since title is used as the commit message which triggers the auto-close) @brandonhaslegs

it doesn't actually, just needs to be in the initial PR description. just added it there :) 

> No no, that's not what I meant! I mean we should definitely give visibility to lesser-known projects by individuals too. If all the featured projects are well known & popular then it would be not as diverse & maybe a bit misaligned. Bdw, I just checked out svelte-stepper & I think it's a cool project that deserve attention!

I got that, all good! I just think graph node should be featured for sure generally (because it's a big name), and am not super sure that featuring a project by one of the core maintainers of Drips is necessarily the best look, though I really appreciate that you like it 🙌 

> Code correct but is stacking 8 projects on mobile too tall?

... We could also just hide everything after the first 4 projects on mobile with a media query. Maybe a bit strange to have different content depending on screen size but I think in this instance it could be OK.

**Comment by SouravInsights** [comment]
I've tried something simple to showcase all the 8 featured projects for smaller screen sizes. Do let me know your feedback! @brandonhaslegs @efstajas 

**Recording:** https://www.loom.com/share/f6be05a3fa7b40df964cdce7b3feb25f?sid=6f64f1bb-2afe-4e77-9c20-512620d4e9fd

**Comment by SouravInsights** [comment]
Also, replaced `svelte-stepper` with `graph-node`

**Review by jo-elimu — APPROVED** [review]

**Review by jo-elimu — CHANGES_REQUESTED** [review]
There is a problem with "Build GQL types":

```
[STARTED] Load GraphQL schemas
[FAILED] 
[FAILED]         Failed to load schema from https://drips-api-mainnet-s1.up.railway.app/:
[FAILED] 
[FAILED]         Context creation failed: Unauthorized
[FAILED]         GraphQLError: Context creation failed: Unauthorized
```

**Comment by efstajas** [comment]
@jo-elimu The reason this happens is because PRs from forked repos cannot access the target repo's secrets. In this case, it's failing, because one of those secrets is the GQL access token.

I'm not super sure how to fix this atm. Looking into it.

**Comment by efstajas** [comment]
Alright, the test thing has bee fixed, tests should pass now 👌

**Review by jo-elimu — APPROVED** [review]

**Comment by SouravInsights** [comment]
Have you guys tested this? Let me know if there's any issues with merging this @efstajas @evvvritt

**Comment by brandonhaslegs** [comment]
Hey, so the overflow for this should be horizontal not vertical.

**Comment by SouravInsights** [comment]
> Hey, so the overflow for this should be horizontal not vertical.

Yeah making it vertically scrollable was intentional.. But if you think horizontal is better then I'll make the necessary changes..

**Comment by brandonhaslegs** [comment]
Yeah I think horizontal scroll is normal behavior, like on Instagram when you post multiple photos

**Review by efstajas — APPROVED** [review]
LGTM. 👍 

Ideally the scrolling would snap to project cards, but we can follow up with this.

**Pull request merged**